climate_scanner/sentiment/playground.py

```diff
@@ -0,0 +1,40 @@
+from sentiment_classifier import *
+
+def get_text():
+    path = get_data('extracted_text.parquet.gzip')
+    text = pd.read_parquet(path).dropna()
```

Comment:
Probably fine for now, but pandas dataframes can be pretty slow when processing large volumes of data.

Reply:
A dictionary might be a better choice here.

Reply:
Assuming this is a test script anyway :)